erpnext_com/www/docs/user/manual/en/setting-up/stock-reconciliation.md

```diff
@@ -180,6 +191,34 @@ As you can see, the Balance Qty as on 10th January got updated from 100 to 120.
 	</iframe>
 </div>
 
+####4.2 For Serialized Items
+
+For item ITEM-00225 has the 6 serial nos HJF00020, HJF00021, HJF00022, HJF00023, HJF00024, HJF00025 with valuation rate as 496.67 per serial no. At the end of the year user has came to know that they have only 3 serial nos against that item with valuation rate 530. So to remove the old serial nos HJF00020, HJF00021, HJF00022, HJF00023, HJF00024, HJF00025 and add the new serial nos with new valuation rate stock reconciliation can be use as follows
```

Comment:
...with new valuation rate, stock reconciliation can be use as follows:
(added comma and semicolon)